test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj

```diff
@@ -41,6 +44,11 @@
 (deftest ^:integration old-master-route-config
   (testing "The old map-style route configuration map still works."
     (bootstrap/with-puppetserver-running-with-mock-jrubies
+     "MOCK.TODO: discuss - this is *probably* ok because all we really care about
+     is that the webserver is accepting connections and handing them to the correct
+     ring handler.  Would be better if we could implement this to hit a clojure
+     endpoint instead of a Ruby one but I don't remember if there are any pure-clojure
+     endpoints under /puppet."
```

Comment:
Do you think it might be possible to do something like add another (optional?) argument to the macro that was just a function that would be called as the implementation of that request handler method in the reify?

Because, for this specific test, if you passed in a function there that returned a 200, I wouldn't have a problem with that.  And I'm sure there are other tests like that.  It just makes me nervous for it to be the default behavior rather than something we have to think about on a test-by-test basis.